js/admin.js

```diff
@@ -342,12 +342,20 @@ var documentsSettings = {
 
 		$(document).on('change', '#secure_view_has_watermark_default_option_cb-richdocuments', function() {
 			var page = $(this).parent();
+
+			// to disable secure-view, print needs to be enabled
 			if (!this.checked) {
-				// to disable secure-view, print needs to be enabled
 				page.find('#secure_view_can_print_default_option_cb-richdocuments')
-					.prop('checked', true);
-				documentsSettings.saveCanPrintDefaultOption(true);
+					.prop('disabled', true);
+			} else {
+				page.find('#secure_view_can_print_default_option_cb-richdocuments')
+					.prop('disabled', false);
 			}
+			page.find('#secure_view_can_print_default_option_cb-richdocuments')
```

Comment:
@jvillafanez yes checkbox that is disabled can be checked/unchecked. Only click() is disabled on that object (this is what disabled stands for). You might have mixed it with hidden

Reply:
It's quite a long time without touching jquery, so maybe it's something they changed at some point? Maybe I'm mistaken... my vote still goes to include a comment for clarification.

Reply:
@jvillafanez so you advise on first checking, and then disabling?

Reply:
For me it's fine if you just add a comment explaining how it will behave and for what reason.
If you're confident it works in all the browsers, a comment will help anyone (me in particular) to know it works without having to check particularly that piece the next time I go over that change.

Reply:
added a comment